mbed_lstools/lstools_base.py

```diff
@@ -206,13 +209,14 @@ def _update_device_from_htm(self, device):
             device.update({"daplink_%s" % f.lower().replace(' ', '_'): v
                            for f, v in daplink_info.items()})
         if htm_target_id:
-            logging.debug("Found htm target id, %s, for usb target id %s",
+            logger.debug("Found htm target id, %s, for usb target id %s",
                             htm_target_id, device['target_id_usb_id'])
             device['target_id'] = htm_target_id
         else:
-            logging.warning("Could not read htm on from usb id %s. "
-                            "Falling back to usb id",
-                            device['target_id_usb_id'])
+            logger.warning("No target ID found inside of the redirect URL in"
+                            " file %s. Falling back to usb id %s",
+                            device['target_id_usb_id'],
+                            join(mount_point, self.MBED_HTM_NAME))
```

Comment:
The supplied strings are in reversed order me thinks

Reply:
Yep.